src/sentry/auth/idpmigration.py

```diff
@@ -15,8 +43,18 @@ def create_verification_key(user: User, org: Organization, email: str) -> None:
     :param org: the organization whose SSO provider is being used
     :param email: the email address associated with the SSO identity
     """
-    key = str(uuid.uuid4())  # noqa
-    raise NotImplementedError  # TODO
+    redis_key = "verificationKeyStorage"
+    TTL = 60 * 10
+    cluster = redis.clusters.get("default").get_local_client_for_key(redis_key)
```

Comment:
Is this using `rb` or `rc`?

Reply:
sorry, could u clarify? im not too familiar with what those mean